### PR TITLE
sending DYK alerts to instructors

### PR DESCRIPTION
**Pull request by Aeropio**
NOTE: Please review the pull request process before opening your first PR: https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/CONTRIBUTING.md#pull-request-process

## What this PR does
It addresses isssue number 2635, DYK alerts now being send to creator of the course/program inaddition to content experts associated with a course.

## Screenshots
Before:
< add a screenshot of the UI before your change >

After:
< add a screenshot of the UI after your change >
## Open questions and concerns
How can i test if DYK alerts are being send to both instructors and content experts in my local machine

### CODE REVIEW

**Comment by bwreid** [comment]
@Aeropio Thanks for the PR. In terms of adding a test, I would look at the `dyk_nominiation_monitor_spec.rb` file. This has a number of tests that should help point you in the right direction.

**Comment by Aeropio** [comment]
@bwreid ok sure i will look into it

**Comment by bwreid** [comment]
@Aeropio It looks like you're encountering a syntax error. I would try running the tests locally so you can more quickly troubleshoot it.

Additionally, we need to add another request as part of this feature change. I'll comment on issue #2635 directly.

**Comment by Aeropio** [comment]
@bwreid I will check for the syntax error and update you

File: package.json [hunk 1]
```diff
@@ -166,7 +166,7 @@
   },
   "devDependencies": {
     "webpack-cli": "^3.3.0",
-    "@babel/plugin-transform-runtime": "^7.3.4"
+    "@babel/plugin-transform-runtime": "^7.4.0"
```
Comment (bwreid) [inline package.json]: I don't think this should be needed as part of this Pull Request

File: app/models/alert.rb [hunk 2]
```diff
@@ -118,6 +118,17 @@ def email_target_user
     update_attribute(:email_sent_at, Time.zone.now)
   end
 
+  def email_course_instructor
+    return if emails_disabled?
+    return if course.nil?
```
Comment (bwreid) [inline app/models/alert.rb]: What about combining this to:
```js
return if emails_disabled? || course.nil?
```

File: spec/lib/alerts/dyk_nomination_monitor_spec.rb [hunk 3]
```diff
@@ -57,6 +62,13 @@ def mock_mailer
       expect(Alert.last.email_sent_at).not_to be_nil
     end
 
+    it 'emails course creator' do
+      create(:courses_user, user_id: course_creator.id, course_id: course.id, role: 1)
+      allow_any_instance_of(AlertMailer).to receive(:alert).and_return(mock_mailer)
+      described_class.create_alerts_for_course_articles
+      expect(Alert.last.email_sent_at).not_to be_nil
```
Comment (bwreid) [inline spec/lib/alerts/dyk_nomination_monitor_spec.rb]: A better check here might be to make sure that the person to whom the email was sent matches up with the `course_creator`. Right now, it looks like you're just checking to make sure an email was sent. I imagine you could either take a look at other properties on the `Alert.last` object or expect `AlertMailer.alert` method to receive the `course_creator`.

**Pull request closed without merging**